src/cdk/testing/harness-environment.ts

```diff
@@ -45,11 +45,16 @@ type ParsedQueries<T extends ComponentHarness> = {
  */
 export abstract class HarnessEnvironment<E> implements HarnessLoader, LocatorFactory {
   // Implemented as part of the `LocatorFactory` interface.
-  rootElement: TestElement;
-
-  protected constructor(protected rawRootElement: E) {
-    this.rootElement = this.createTestElement(rawRootElement);
+  get rootElement(): TestElement {
+    this._rootElement = this._rootElement || this.createTestElement(this.rawRootElement);
+    return this._rootElement;
+  }
+  set rootElement(element: TestElement) {
```

Comment:
Not a fan of this getter/setter, but the problem is that this class calls `createTestElement` in the constructor before the sub-classes have had a change to assign the stabilization method.